public/apps/configuration/panels/role-edit.tsx

```diff
@@ -13,19 +13,27 @@
  *   permissions and limitations under the License.
  */
 
-import React from 'react';
-import { AppDependencies } from '../../types';
-import { PanelWithHeader } from '../utils/panel-with-header';
 import {
+  EuiButton,
+  EuiComboBox,
+  EuiComboBoxOptionOption,
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiLink,
   EuiPageHeader,
+  EuiSpacer,
   EuiText,
   EuiTitle,
-  EuiLink,
-  EuiForm,
-  EuiFormRow,
-  EuiFieldText,
-  EuiSpacer,
 } from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import { AppDependencies } from '../../types';
+import { CLUSTER_PERMISSIONS, INDEX_PERMISSIONS } from '../constants';
+import { fetchActionGroups } from '../utils/action-groups-utils';
+import { FormRow } from '../utils/form-row';
```

Comment:
nit. mixed naming convention

Reply:
Is there guide for the naming convention?  I try to do upper camel case for classes (though it is also a function now) and lower camel case for function.

Reply:
fyi
https://github.com/felixge/node-style-guide#use-uppercase-for-constants

Reply:
It says lower camel case for function and upper camel case for class. Here `fetchActionGroups` is a function, `FormRow` is a class and `CLUSTER_PERMISSIONS, INDEX_PERMISSIONS` are constants. So they match the convention, do they?

Reply:
My bad, looks good, thanks 👍